be/src/connector/jdbc_connector.cpp

```diff
@@ -51,6 +51,10 @@ static std::string get_jdbc_sql(const Slice jdbc_url, const std::string& table,
                                 const std::vector<std::string>& filters, int64_t limit) {
     std::ostringstream oss;
     oss << "SELECT";
+    if (limit != -1 && jdbc_url.starts_with("jdbc:sqlserver")) {
```

Comment:
i think for sqlserver the sql will be select top(xx) xxx from xxx limit xxx;

Reply:
i set the limit=-1, this code  "if (limit != -1) {
        if (jdbc_url.starts_with("jdbc:oracle")) {
            // oracle doesn't support limit clause, we should generate a subquery to do this
            // ref: https://stackoverflow.com/questions/470542/how-do-i-limit-the-number-of-rows-returned-by-an-oracle-query-after-ordering
            return fmt::format("SELECT * FROM ({}) WHERE ROWNUM <= {}", oss.str(), limit);
        } else {
            oss << " LIMIT " << limit;
        }
    }"   is no longer valid ?

Reply:
oh, got it.